src/forms/BoundDateField.test.tsx

```diff
@@ -36,7 +37,7 @@ describe("BoundDateField", () => {
     const r = await render(<BoundDateField field={author.birthday} />);
 
     // When triggering the Date Picker
-    r.birthday().focus();
+    focus(r.birthday);
```

Comment:
**thought:** I am a little concerned about using `focus` here instead of `.focus` as I believe they have different behaviors. Specifically:
`element.focus()`: Actually applies focus to this element in the DOM, which in turn triggers the "focus" event.
`fireEvent.focus(element)`: Does not apply focus to the element, only triggers the focus event as if the element _was_ put in focus.

That nuance can be a bit of a pain to figure out. But, if everything still passes, then great. I am just kind of surprised.

Reply:
Yeah, fwiw I'd like for our `focus(...)` to eventually more fully emulate `element.focus()`, but for now `focus(...)` a) works, and b) does the `act` for us, otherwise would need to write:

```
act(() => r.birthday.focus());
```

So, yeah, since it passes the test, seems fine for now.